examples/README.md

```diff
@@ -1,233 +0,0 @@
-# Running aws-operator Locally
```

Comment:
I removed whole examples directory. There are outdated charts, and no one cares about them.